e2e/README.md

```diff
@@ -205,7 +205,7 @@ Our file structure follows the type of type of testing being excercised at the e
 |`./appActions.js`             | Contains common methods which can be leveraged by test case authors to quickly move through the application when writing new tests.|
 |`./baseFixture.js`            | Contains base fixtures which only extend default `@playwright/test` functionality. The expectation is that these fixtures will be removed as the native Playwright API improves|
 
-Our functional tests end in `*.e2e.spec.js`, visual tests in `*.visual.spec.js` and performance tests in `*.perf.spec.js`.
+Our functional tests end in `*.e2e.spec.js`, visual tests in `*.visual.spec.js` and performance tests in `*.perf.spec.mjs`.
```

Comment:
Do the other test types need updated here as well?

Reply:
Ooh good catch, this is actually a typo